src/Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
```

Comment:
Just so I can understand - does this props file get auto-imported somehow by all other projects? I don't understand how this part works so would be nice to understand.

Reply:
yes, Directory.Build.props and Directory.Build.targets are automatically included by msbuild/dotnet sdk.